tests/double_require.rs

```diff
@@ -25,7 +25,7 @@ static ONLY_C_ERROR: &str = "error: The following required arguments were not pr
     -b
 
 USAGE:
-    prog [OPTIONS] -b -c
+    prog [OPTIONS] -c -b
```

Comment:
The changes in this file are intentional because the earlier messages were wrong. We always show existing arguments first in other cases too.